utils/s2n_set.c

```diff
@@ -72,9 +72,9 @@ static S2N_RESULT s2n_set_binary_search(struct s2n_set *set, void *element, uint
     return S2N_RESULT_OK;
 }
 
-struct s2n_set *s2n_set_new(size_t element_size, int (*comparator)(const void*, const void*))
+struct s2n_set *s2n_set_new(uint32_t element_size, int (*comparator)(const void*, const void*))
 {
-    ENSURE_REF_PTR(comparator);
+    if (comparator == NULL) return NULL;
```

Comment:
Why the change here?

Reply:
I replaced it with `notnull_check_ptr` and I'm tracking this change with issue https://github.com/awslabs/s2n/issues/2252. I'll revert the changes in `notnull_check_ptr` once this issue is solved.